Pony/include/pony/Socket.hpp

```diff
@@ -6,36 +6,64 @@
 /* Original story "Networking for Game Programmers" by Glenn Fiedler
  */
 
-#ifndef PONY_SOCKET_HPP
-#define PONY_SOCKET_HPP
+#ifndef ___PONY__SOCKET_HPP___
+#define ___PONY__SOCKET_HPP___
 
 #include <pony/Address.hpp>
-#include <pony/SharedMacros.hpp>
+
+#if defined(_WIN32)
+    #include <winsock2.h>
+    #pragma comment( lib, "ws2_32.lib" )
+#endif
 
 namespace pony
 {
 
-class PONY_EXPORT Socket
+///HACK: to make Winsock happy we need in socketInit()/Kill()!
```

Comment:
hack ??
also I prefer no jokes in comments

Reply:
Better use Doxygen's `@note` to write such comments.

Reply:
but this is true - we need this couple of function just to make Windows runtime happy!! just delete this comment - ok ?